### ISSUE TITLE
dnstt needs PT spec 1.0 support for Tor Project to accept it

### ISSUE THREAD

**Issue by tladesignz** [tladesignz/IPtProxy#24]
- Fork dnstt
- Add [goptlib](https://gitweb.torproject.org/pluggable-transports/goptlib.git/)
- Rework, so configuration is accepted via goptlib.

**Comment by n8fr8** [comment]
Yes, we should move forward with this. We want DNSTT to be usable by those tools that still require the PT 1.0 spec.

**Comment by tladesignz** [comment]
First stab at adding PT 1 compatibility:
https://github.com/tladesignz/dnstt/commit/ef2274719351093bc20fbe912608536f89348d63

336a6b7e28edeaa2c9546cfeafd0b815072515c3

**Comment by tladesignz** [comment]
Also announced here:
https://gitlab.torproject.org/tpo/anti-censorship/pluggable-transports/trac/-/issues/40001#note_2809890

**Comment by tladesignz** [comment]
Progress and discussion here:
https://gitlab.torproject.org/tpo/anti-censorship/pluggable-transports/trac/-/issues/40001#note_2809890

**Comment by tladesignz** [comment]
Work can be found here:
https://github.com/tladesignz/IPtProxy/tree/dnstt

and here:
https://github.com/tladesignz/dnstt

Seems to to be finished. Waiting on feedback from Tor Project.
